packages/sveltekit/src/server/handle.ts

```diff
@@ -106,87 +19,7 @@ export function addSentryCodeToPage(options: SentryHandleOptions): NonNullable<R
  * ```
  */
 export function sentryHandle(handlerOptions?: SentryHandleOptions): Handle {
-  const options = {
-    handleUnknownRoutes: false,
-    injectFetchProxyScript: true,
-    ...handlerOptions,
-  };
-
-  const sentryRequestHandler: Handle = input => {
-    // event.isSubRequest was added in SvelteKit 1.21.0 and we can use it to check
-    // if we should create a new execution context or not.
-    // In case of a same-origin `fetch` call within a server`load` function,
-    // SvelteKit will actually just re-enter the `handle` function and set `isSubRequest`
-    // to `true` so that no additional network call is made.
-    // We want the `http.server` span of that nested call to be a child span of the
-    // currently active span instead of a new root span to correctly reflect this
-    // behavior.
-    // As a fallback for Kit < 1.21.0, we check if there is an active span only if there's none,
-    // we create a new execution context.
-    const isSubRequest = typeof input.event.isSubRequest === 'boolean' ? input.event.isSubRequest : !!getActiveSpan();
-
-    if (isSubRequest) {
-      return instrumentHandle(input, options);
-    }
-
-    return withIsolationScope(isolationScope => {
-      // We only call continueTrace in the initial top level request to avoid
-      // creating a new root span for the sub request.
-      isolationScope.setSDKProcessingMetadata({
-        normalizedRequest: winterCGRequestToRequestData(input.event.request.clone()),
-      });
-      return continueTrace(getTracePropagationData(input.event), () => instrumentHandle(input, options));
-    });
-  };
+  const sentryRequestHandler = sentryHandleGeneric(continueTrace, handlerOptions);
```

Comment:
Same question here: can we just re-export `sentryHandleGeneric` as `sentryHandle`?

Reply:
See my other reply for more detail. This avoids a breaking change, and avoids exposing the implementation detail of which precise `continueTrace` function is used.